VideoLocker/src/main/java/org/edx/mobile/view/common/MessageType.java

```diff
@@ -1,8 +1,5 @@
 package org.edx.mobile.view.common;
 
-/**
- * Created by hanningni on 8/17/15.
- */
 public enum MessageType {
-    FLYIN_ERROR, FLYIN_WARNING, FLYIN_INFO, ERROR, WARNING, INFO, EMPTY
+    FLYIN_ERROR, FLYIN_WARNING, FLYIN_INFO, ERROR, WARNING, INFO, EMPTY, PERSISTENT_ERROR
```

Comment:
Instead of `PERSISTENT_ERROR`, I suggest just naming it `DIALOG`, as the other error message platforms may be persistent too.